apps/api/src/app/events/usecases/cancel-delayed/cancel-delayed.usecase.ts

```diff
@@ -34,6 +37,71 @@ export class CancelDelayed {
       }
     );
 
+    const mainDigestJob = transactionJobs.find((job) => isMainDigest(job.type, job.status));
```

Comment:
Here i want to point to the naming issue I had, i am not entirely sure what to call the digest that is responsible for executing the digest in the end of the digest time.

At the current state, I referred to this digest job as `main`, and if we would find a new transaction that will place it a `follower`.

If someone has any other ideas I'd love to hear them.